Casks/l/lyricfever.rb

```diff
@@ -0,0 +1,21 @@
+cask "lyricfever" do
```

Comment:
```suggestion
cask "lyric-fever" do
```

We should change the token to `lyric-fever` as per our [token guidelines](https://docs.brew.sh/Cask-Cookbook#converting-the-simplified-name-to-a-token). The filename will also need to be updated.